Nagstamon/Servers/Generic.py

```diff
@@ -1088,7 +1088,8 @@ def GetStatus(self, output=None):
                             self.Debug(server=self.get_name(),
                                        debug='Filter: SOFT STATE ' + str(host.name) + ';' + str(service.name))
                         service.visible = False
-                else:
+                # fix for https://github.com/HenriWahl/Nagstamon/issues/654
+                elif not self.TYPE.startswith("Zabbix"):
```

Comment:
Can you find a solution without changing Generic.py to keep it cleaner between the several server implementations?

Reply:
Filtering in Generic.py (checking against configured filters and acknowledged/scheduled_downtime then setting visible to False) is done above this check against length of attempt. There literary not much more that can be done inside Zabbix.py that would be much more cleaner.
I can think of two different solutions:

1. Implement another filtering inside Zabbix.py - but that just redundant 
2. Pad every attempt with white spaces to be more then 3 characters in length - ugly hack.

Reply:
Well, I see... and anyway there are a bunch of server-specific conditions already in there. So it may be.

Hope one day find the time to clean all this old code up.